CMake/unix_config.cmake

```diff
@@ -10,7 +10,6 @@ macro(os_set_flags)
     if(${MACHINE} MATCHES "arm-*")
         set(CMAKE_C_FLAGS   "${CMAKE_C_FLAGS}   -mfpu=neon -mfloat-abi=hard -ftree-vectorize -latomic")
         set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -mfpu=neon -mfloat-abi=hard -ftree-vectorize -latomic")
-        add_definitions(-DRASPBERRY_PI)
```

Comment:
Pls verify that there are no other places that use this DEFINE, including Android.

Reply:
The string "RASPBERRY_PI" is not found anywhere in the code.